getting_hired/applying_and_interviewing/applying.md

```diff
@@ -2,12 +2,8 @@
 
 The next step is to actually apply.  Don't send all your applications out at once! You'll get totally overwhelmed if you start hearing back from people and need to complete tasks all at once.  Treat it like an iterative process where you'll send out several applications each day, learn from any mistakes you can identify based on feedback from the process, and try again the next day.  
 
-<div class="lesson-note" markdown="1>
```

Comment:
Thanks for spoting this.

The div here is intentional which will provide a gold border to the content inside it. This was [recently added to our website](https://github.com/TheOdinProject/theodinproject/pull/3032). 

But I suppose it didn't render correctly because of the missing quote in `markdown="1`. Are you willing to fix this so that the quote wraps the 1 like so: `markdown="1"`?

Reply:
Now I got it. Sure, I'll fix it right now.